### PR TITLE
Update chocolatey to the latest version

### PR DESCRIPTION
**Pull request by j4ckofalltrades**
# Pull Request Description

This pull request relates to strongbox/strongbox#1826

# Acceptance Test

* [x] Running `./build.sh` successfully builds all images.

# Questions

* Does this pull request somehow break backward compatibility? 
  * [ ] Yes, it will require fixing build jobs.
  * [x] No

* Does this pull request require other pull requests to be merged first? 
  * [ ] Yes, please see #...
  * [x] No

* Does this require an update of the documentation?
  * [ ] Yes, please see strongbox/strongbox-docs#{PR_NUMBER}
  * [x] No

### CODE REVIEW

File: images/alpine/jdk11/Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget3.4-choco0.10 [hunk 1]
```diff
@@ -15,6 +15,8 @@ ENV ChocolateyInstall=/opt/chocolatey
 
 COPY common-scripts /scripts
 COPY --from=builder /usr/src/choco/build_output/chocolatey $ChocolateyInstall
+# see https://github.com/mono/mono/issues/12557
```
Comment (j4ckofalltrades) [inline images/alpine/jdk11/Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget3.4-choco0.10:18]: Fix for error that @steve-todorov mentioned in https://github.com/strongbox/ci-build-images/pull/17

**Comment by j4ckofalltrades** [comment]
@steve-todorov Requesting review

File: images/alpine/jdk11/Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget3.4-choco0.10 (hunk 1 above)
Comment (steve-todorov) [reply images/alpine/jdk11/Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget3.4-choco0.10:18]: Thanks for mentioning it in a comment! :)

**Review by steve-todorov — APPROVED** [review]
LGTM! Many thanks for fixing the error as well! :)

**Comment by carlspring** [comment]
Excellent work, thanks a million! :)

**Pull request merged**